src/cow-react/modules/wallet/web3-react/connectors/Injected/index.tsx

```diff
@@ -0,0 +1,204 @@
+import {
```

Comment:
is this PR adding all this? i assume you just have the wrong base-branch, right?

Reply:
Yes, because we are adding here a custom injected connector that we can adjust for different wallets, mostly to fix some specific bugs each wallet introduces.

Reply:
Okay, if this is new code, do you think you give enough context to understand it? 

All you say is `Adds a custom injected connector` which falls short forthis 204 of code.

Some of the questions I have:
- Why did you need to do this (there's an injected connector already in the library)
- Did you write all this code or C&P most of it 
- If this is Copy Pasted from somewhere, where from? can anyone reading the code and PR description get this info
- What parts are modified from this original code? And why?
- Was the only alternative to modify the behavior? Wasn't it possible to, for example, use inheritance or composition for this?

Reply:
It is based on https://github.com/Uniswap/web3-react/blob/main/packages/metamask/src/index.ts
I've added comments which parts are copied, custom or modified.
Its probably possible to use inheritance or composition for this, but for the sake of time I think we can do this for now and update it in the future.